### PR TITLE
Fix regression caused by upgrade of Azure API libs

### PR DESCRIPTION
**Pull request by asmorodskyi**

### CODE REVIEW

File: Dockerfile_dev [hunk 1]
```diff
@@ -10,14 +10,8 @@ COPY requirements.txt /tmp/
...
 WORKDIR /pcw
 
-## Finalize ################################################################# ##
-
-VOLUME /pcw/db
-
-EXPOSE 8000/tcp
-
-ENTRYPOINT ["sh", "-c", "/pcw/container-startup", "$@"]
+ENTRYPOINT ["sh", "-c", "/pcw/container-startup updaterun"]
```
Comment (pdostal) [inline Dockerfile_dev:17]:
```suggestion
ENTRYPOINT ["sh", "-c", "/pcw/container-startup"]
```
I don't see a reason for this argument in `ENTRYPOINT`. I think that it makes sense to use this container file like:
```bash
podman build -t pcw_dev .
podman run -it --rm pcw_dev updaterun
podman run -it --rm pcw_dev removeclusters
podman run -it --rm pcw_dev orderquatroformagipizza
```

Comment (asmorodskyi) [reply Dockerfile_dev:17]: the problem that it does not work like that : 
```
podman run  -v /etc/pcw.ini:/etc/pcw.ini -v /var/pcw/:/var/pcw -v /home/asmorodskyi/source/pcw/:/pcw/  -t pcw-devel updaterun                       [15:11:27]
pcw startup helper
Usage: /pcw/container-startup COMMAND
COMMANDS:
   help                    Print help message
   check                   Run system health checks
   run                     Standart publiccloud-watcher run procedure
   migrate                 Run database migration
   createsuperuser         Create superuser. Requires --email and --username argument
	 updaterun 							 Do a regular instance scan and persist results to local DB
	 rmclusters							 Scan whole AWS account and delete ANY EKS clusters

All other commands will be passed as arguments to 'manage.py'
```
 ( when I apply your suggestion and re-build the container )

Comment (grisu48) [reply Dockerfile_dev:17]: Pavel suggestion works, you only need to provide the additional commands as argument in your `podman run` command just as he suggested:

    podman build -t pcw_dev .
    podman run -it --rm pcw_dev updaterun
    podman run -it --rm pcw_dev removeclusters
    podman run -it --rm pcw_dev orderquatroformagipizza

Also :+1: for the :pizza: @pdostal :wink: Me gusta!

Comment (asmorodskyi) [reply Dockerfile_dev:17]: why is not working for me ? 
```
podman run  -v /etc/pcw.ini:/etc/pcw.ini -v /var/pcw/:/var/pcw -v /home/asmorodskyi/source/pcw/:/pcw/  -t pcw-devel updaterun                       [15:11:27]
pcw startup helper
Usage: /pcw/container-startup COMMAND
COMMANDS:
   help                    Print help message
   check                   Run system health checks
   run                     Standart publiccloud-watcher run procedure
   migrate                 Run database migration
   createsuperuser         Create superuser. Requires --email and --username argument
	 updaterun 							 Do a regular instance scan and persist results to local DB
	 rmclusters							 Scan whole AWS account and delete ANY EKS clusters

All other commands will be passed as arguments to 'manage.py'
```

again I am running "podman run  -v /etc/pcw.ini:/etc/pcw.ini -v /var/pcw/:/var/pcw -v /home/asmorodskyi/source/pcw/:/pcw/  -t pcw-devel *updaterun*"

**Review by pdostal — APPROVED** [review]

**Pull request merged**